docs/core/install/linux-centos.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Install .NET Core on CentOS - .NET Core
+description: Demonstrates the various ways to install .NET Core SDK and .NET Core Runtime on CentOS.
+author: thraka
+ms.author: adegeo
+ms.date: 06/04/2020
+---
+
+# Install .NET Core SDK and .NET Core Runtime on CentOS
+
+.NET Core is supported on CentOS. This article describes how to install .NET Core on CentOS.
+
+[!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
+
+[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+
+## Supported distributions
+
+The following table is a list of currently supported .NET Core releases on both CentOS 7 and CentOS 8. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of CentOS is no longer supported.
+
+- A ✔️ indicates that the version of CentOS or .NET Core is still supported.
+- A ❌ indicates that the version of CentOS or .NET Core isn't supported on that CentOS release.
+- When both a version of CentOS and a version of .NET Core have ✔️, that OS and .NET combination are supported.
+
+| CentOS                   | .NET Core 2.1 | .NET Core 3.1 | .NET 5 Preview |
+|--------------------------|---------------|---------------|----------------|
+| ✔️ [8](#centos-8-) | ✔️ 2.1        | ✔️ 3.1        | ✔️ 5.0 Preview |
+| ✔️ [7](#centos-7-) | ✔️ 2.1        | ✔️ 3.1        | ✔️ 5.0 Preview |
+
+The following versions of .NET Core are no longer supported. The downloads for these still remain published:
+
+- 3.0
+- 2.2
+- 2.0
+
+[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+
+## How to install other versions
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
+
+## CentOS 8 ✔️
```

Comment:
Also, not sure I like icons in the headings.

Reply:
Me neither, but some headings were really long and introduced wrapping on smaller screens if we but `(supported)` or `(not supported)` it looked terrible.